dev-python/pymupdf/pymupdf-1.19.6.recipe

```diff
@@ -0,0 +1,77 @@
+SUMMARY="Python bindings for MuPDF's rendering library"
+DESCRIPTION="PyMuPDF is a Python binding with support for MuPDF, a lightweight PDF, XPS, and \
+E-book viewer, renderer, and toolkit, which is maintained and developed by Artifex Software, Inc.
+MuPDF can access files in PDF, XPS, OpenXPS, CBZ, EPUB and FB2 (e-books) formats, and it is known \
+for its top performance and high rendering quality.
+With PyMuPDF you can access files with extensions like \".pdf\", \".xps\", \".oxps\", \".cbz\", \
+\".fb2\" or \".epub\". In addition, about 10 popular image formats can also be handled like \
+documents: \".png\", \".jpg\", \".bmp\", \".tiff\", etc.."
+HOMEPAGE="https://pypi.org/project/PyMuPDF/
+	https://github.com/pymupdf/PyMuPDF/"
+COPYRIGHT="2022 Jorj X. McKie and Ruikai Liu"
+LICENSE="AGPL-3.0"
+REVISION="1"
+pypi="032d24e0c774e67742395fda163a172c60e4d0f9875785d5199eb2956d5e"
+SOURCE_URI="https://files.pythonhosted.org/packages/9f/1d/$pypi/PyMuPDF-$portVersion.tar.gz"
+CHECKSUM_SHA256="ef3d13e27f1585d776f6a2597f113aabd28d36b648b983a72850b21c5399ab08"
+SOURCE_DIR="PyMuPDF-$portVersion"
+PATCHES="pymupdf-$portVersion.patchset"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	$portName = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libmupdf$secondaryArchSuffix
+	lib:libfreetype$secondaryArchSuffix
```

Comment:
Both `libmupdf` and `libfreetype` are linked in the `pythonPackages` so they should be moved to `REQUIRES_$pythonPackage`
I had pymupdf_python3 installed when I removed mupdf, it didn't ask me to remove pymupdf_python3, atm only the base (empty) package require both libs which isn't needed.

Reply:
https://github.com/haikuports/haikuports/pull/6825#pullrequestreview-943304456

Reply:
Holding off on this until we decide what to do about versioning